plugin-cgroupv2/src/cgroupv2.rs

```diff
@@ -55,15 +76,43 @@ impl FromStr for CgroupV2Metric {
 
 #[derive(Clone)]
 pub struct Metrics {
+    /// Total CPU usage time by the cgroup.
     pub time_used_tot: TypedMetricId<u64>,
+    /// CPU in user mode usage time by the cgroup.
     pub time_used_user_mode: TypedMetricId<u64>,
+    /// CPU in system mode usage time by the cgroup.
     pub time_used_system_mode: TypedMetricId<u64>,
+    /// Anonymous used memory, corresponding to running process and various allocated memory.
+    pub anon_used_mem: TypedMetricId<u64>,
+    /// Files memory, corresponding to open files and descriptors.
+    pub file_mem: TypedMetricId<u64>,
+    /// Memory reserved for kernel operations.
+    pub kernel_mem: TypedMetricId<u64>,
+    /// Memory used to manage correspondence between virtual and physical addresses.
+    pub pagetables_mem: TypedMetricId<u64>,
+    /// Total memory used by cgroup.
+    pub total_mem: TypedMetricId<u64>,
 }
 
 impl Metrics {
+    /// Provides a information base to create metric before sending CPU and memory data,
+    /// with `name`, `unit` and `description` parameters.
+    ///
+    /// # Arguments
+    ///
+    /// * `alumet` - A AlumetPluginStart structure passed to plugins for the start-up phase.
+    ///
+    /// # Returns
+    ///
+    /// * `Result` - Type representing success `Ok` or failure `Err`.
+    /// * `MetricCreationError` - Error which can occur when creating a new metric.
+    ///
     pub fn new(alumet: &mut AlumetPluginStart) -> Result<Self, MetricCreationError> {
-        let usec: PrefixedUnit = PrefixedUnit::micro(Unit::Second);
+        let usec = PrefixedUnit::micro(Unit::Second);
+        let raw = Unit::Byte;
```

Comment:
Why `raw`? With a simple unit like that, don't even bother creating a local variable and cloning it, use `Unit::Byte` directly.